app/assets/javascripts/redux/modules/models/responses.js

```diff
@@ -64,4 +65,15 @@ export const reducer = handleActions({
 
   [LISTEN_SIGNALS_POST_REQUEST_SUCCESS]: handleResponseRequest,
 
+  [LISTEN_SIGNALS_DELETE_REQUEST_SUCCESS]: (state, action) => {
+    const responses = action.meta.signal.responses;
```

Comment:
Ah, so I did try doing meta.signal.responses but it returns an array of two response objects.

Our payload vs how we normalize is different since we pass signal to create..